risc0/zkvm/platform/src/syscall.rs

```diff
@@ -690,6 +690,7 @@ pub unsafe extern "C" fn sys_argv(
 }
 
 #[cfg_attr(feature = "export-syscalls", no_mangle)]
+#[deprecated]
```

Comment:
small nit: could we provide a message to offer an alternative function for them to use? https://doc.rust-lang.org/nightly/reference/attributes/diagnostics.html#the-deprecated-attribute

Reply:
There's no alternative, it shouldn't have been made a public syscall to begin with (IMHO). Also, too late but we should not have `sys_alloc_aligned` either as a syscall.